alchemy/src/util/telemetry/client.ts

```diff
@@ -54,13 +54,9 @@ export class TelemetryClient implements ITelemetryClient {
   }
 
   record(event: Telemetry.EventInput, timestamp = Date.now()) {
-    if (!this.context) {
-      return;
-    }
```

Comment:
Why did this change? Looks like we have a `this.context!` below which is dangerous and could crash?

Reply:
ohhh wait no I remember. because context is not on every event anymore. this check should've been moved to `send` not removed. add the check back on `send`